src/shared/components/palette/datetime-picker/datetime-picker.component.ts

```diff
@@ -127,7 +127,7 @@ export class DatetimePickerComponent extends AbstractFormFieldComponent implemen
       this.hideTime = true;
       this.checkTime = false;
       this.dateTimeEntryFormat = this.formatTranslationService.removeTime(this.dateTimeEntryFormat);
-      this.momentFormat = 'YYYY-MM-DD';
+      this.momentFormat = 'YYYY-MM-DD'
```

Comment:
Removed the semicolon why?

Reply:
that was a mistake i think with the merge. will put the semi colon back